packages/neuron-ui/src/components/SUDTReceive/index.tsx

```diff
@@ -13,18 +15,33 @@ import { RoutePath, CONSTANTS } from 'utils'
 import styles from './sUDTReceive.module.scss'
 
 const { DEFAULT_SUDT_FIELDS } = CONSTANTS
+const toShortAddr = (addr: string) => {
```

Comment:
This function looks almost the same as `toShortAddr` at `Receive/index/tsx`

Reply:
I left them in component modules respectively because they are simple and will be removed eventually. Without sharing them between components makes sure no codes will be left on removing.